bin/infrasim-main

```diff
@@ -60,6 +60,16 @@ if __name__ == '__main__':
             print "Restart InfraSIM service..."
             socat.start_socat()
             ipmi.start_ipmi()
+        elif sys.argv[1] == "version":
+            qemu_ver_cmd = qemu.get_qemu() + " --version"
+            ipmi_ver_cmd = ipmi.get_ipmi() + " -v"
+            socat_ver_cmd = socat.get_socat() + " -V"
+            print "Kernel:  ", run_command("uname -sr")[1].split('\n')[0]
+            print "Base OS: ", run_command("cat /etc/issue")[1].split('\\')[0]
+            print "QEMU:    ", run_command(qemu_ver_cmd)[1].split(',')[0]
+            print "OpenIPMI:", run_command_anyway(ipmi_ver_cmd)[1].split('\n')[0]
+            print "Socat:   ", ' '.join(run_command(socat_ver_cmd)[1].split('\n')[1].split(' ')[0:3])
+            print "InfraSIM:"
```

Comment:
last version should be something like:

infrasim-compute: a.b.c

@XiaowenJiang could you add a mechanism define version number of infrasim-compute? And then it can be displayed here.